cluster/ci/monitoring/canary.json

```diff
@@ -0,0 +1,171 @@
+{
+   "__inputs": [ ],
+   "__requires": [ ],
+   "annotations": {
+      "list": [ ]
+   },
+   "editable": false,
+   "gnetId": null,
+   "graphTooltip": 0,
+   "hideControls": false,
+   "id": null,
+   "links": [ ],
+   "panels": [
+      {
+         "aliasColors": { },
+         "bars": false,
+         "dashLength": 10,
+         "dashes": false,
+         "datasource": "prometheus",
+         "description": "sum(rate(prowjob_state_transitions{job=\"plank\",job_name=~\"canary-openshift-ocp-installer-e2e-.*-4.2\",job_name!~\"rehearse.*\",state=\"success\"}[48h]))/sum(rate(prowjob_state_transitions{job=\"plank\",job_name=~\"canary-openshift-ocp-installer-e2e-.*-4.2\",job_name!~\"rehearse.*\",state=~\"success|failure\"}[48h]))",
+         "fill": 1,
+         "gridPos": {
+            "h": 9,
+            "w": 24,
+            "x": 0,
+            "y": 0
+         },
+         "id": 2,
+         "legend": {
+            "alignAsTable": true,
+            "avg": false,
+            "current": true,
+            "max": false,
+            "min": true,
+            "rightSide": true,
+            "show": true,
+            "sideWidth": 350,
+            "sort": "min",
+            "sortDesc": true,
+            "total": false,
+            "values": true
+         },
+         "lines": true,
+         "linewidth": 1,
+         "links": [ ],
+         "nullPointMode": "null",
+         "percentage": false,
+         "pointradius": 5,
+         "points": false,
+         "renderer": "flot",
+         "repeat": null,
+         "seriesOverrides": [ ],
+         "spaceLength": 10,
+         "stack": false,
+         "steppedLine": false,
+         "targets": [
+            {
+               "expr": "sum(rate(prowjob_state_transitions{job=\"plank\",job_name=~\"canary-openshift-ocp-installer-e2e-aws-fips.*-4.2\",job_name!~\"rehearse.*\",state=\"success\"}[48h]))/sum(rate(prowjob_state_transitions{job=\"plank\",job_name=~\"canary-openshift-ocp-installer-e2e-aws-fips.*-4.2\",job_name!~\"rehearse.*\",state=~\"success|failure\"}[48h]))",
+               "format": "time_series",
+               "intervalFactor": 2,
+               "legendFormat": "AWS IPI FIPS",
```

Comment:
A Question:
Is there `AWS IPI` without `FIPS`?

Reply:
Yes, but that is an existing release blocker. This dashboard is for the non-blocking release informers.

Reply:
Ack. Blaming myself about missing the background knowledge.